api-cursors/src/main/java/org/zalando/nakadi/service/CursorsService.java

```diff
@@ -142,11 +142,22 @@ private void validateStreamId(final List<NakadiCursor> cursors,
             }
         }
     }
+    public List<SubscriptionCursorWithoutToken> getSubscriptionCursorsForUpdate(final String subscriptionId)
```

Comment:
This method is not needed, as the value to update is stored in zk, so the only case to take it from db is to ensure that subscription still exists. 
As it still adds value (check subscription existence), then let's leave it as it is